mmv1/third_party/terraform/website/docs/r/storage_bucket.html.markdown

```diff
@@ -69,6 +69,28 @@ resource "google_storage_bucket" "auto-expire" {
 }
 ```
 
+## Example Usage - Life cycle settings for storage bucket objects with `no_age` enabled
+When creating a life cycle condition that does not also include an `age` field, it's important to include the `no_age` flag so your new bucket does not unintentionally create an extreanous condition that may interact incorrectly with your other options.
+
+```hcl
+resource "google_storage_bucket" "no-age-enabled" {
+  provider = google-beta
+  name          = "no-age-enabled-bucket"
+  location      = "US"
+  force_destroy = true
+
+  lifecycle_rule {
+    action {
+      type = "Delete"
+    }
+    condition {
+      days_since_noncurrent_time = 3
+      no_age = true
+    }
+  }
+}
+```
+
 ## Example Usage - Enabling public access prevention
 
 ```hcl
```

Comment:
Can we also update the documentation for `age` below to explicitly mention the default of `0`? "In a lifecycle_rule block where this is not set, it defaults to 0 unless no_age is set to true" or similar

Actually, let's make the other fields explicit about zero values as well. "Due to a bug, this cannot currently be set to 0. When set to that value in configuration, it will be ignored and treated as if there was no condition instead."